instances.go

```diff
@@ -92,6 +93,17 @@ type InstanceBackup struct {
 	} `json:"schedule,omitempty"`
 }
 
+type instanceDiskEncryption string
+
+func (i *instanceDiskEncryption) String() string {
+	return string(*i)
+}
+
+var (
```

Comment:
Can we make them as `const`?

```suggestion
const (
```

Reply:
Done